csp_billing_adapter/config.py

```diff
@@ -17,40 +17,68 @@
 import yaml
 
 
-class Config:
+class Config(dict):
+    """The Config class handles reading configuartion settings
+    from a provided configuration file and loading default settings from
+    any configured plugins  This class is based on a dict to allow access
+    using both dot notation and key lookup.
+
+    For example:
+
+        result = Config(config_data)
+        print(result.get('Name'))
+        print(result.Name)
+
```

Comment:
Extraneous blank line at end of doc string.

Reply:
fixed